apps/ewallet/lib/ewallet/fetchers/transaction_consumption_fetcher.ex

```
@@ -23,16 +23,16 @@ defmodule EWallet.TransactionConsumptionFetcher do
 
   @spec get(String.t()) ::
           {:ok, %TransactionConsumption{}}
-          | {:error, :transaction_consumption_not_found}
-  def get(nil), do: {:error, :transaction_consumption_not_found}
+          | {:error, :invalid_parameter}
+  def get(nil), do: {:error, :invalid_parameter}
```

Can we also return an error description e.g. ``"`id` cannot be nil"``?